### ISSUE TITLE
Rename pattern provider from its own gui or even from the pattern access terminal

### ISSUE THREAD

**Issue by Technici4n** [AppliedEnergistics/Applied-Energistics-2#6985]
Self-explanatory. Naming PPs in an anvil or with a name press is too tedious.

**Comment by AlisonHuang777** [comment]
Will this get implemented any time soon? It's been a year.

**Comment by MalcolmAnderson** [comment]
Agreed, it would be nice to be able to know "which chest is attached to which machine?"
Or, I have 3 "molecular assembler 8" 's

**Comment by peter-ahe** [comment]
If you have the quartz cutting knife in your main hand and click on a pattern provider, you can rename it immediately.

![2024-08-20_08 50 56](https://github.com/user-attachments/assets/0e6a9d4a-33bc-4f01-a870-f76551f92bb9)
